### ISSUE TITLE
Removing unused imported name leaves double space inside braces

### ISSUE THREAD

**Issue by quezak** [sweepline/eslint-plugin-unused-imports#6]
Using the `unused-imports/no-unused-imports-ts` rule, plugin version 0.1.2. Please let me know if I need to provide full config or some other data.

This happens if the removed item is the first or the middle one.
If the last item is unused, it's removed correctly (trailing comma removed and no double space) -- but maybe that's the result of my other rules fixing the trailing comma?

Example 1:
```typescript
// file:
import { IsInt, IsOptional, IsString } from 'class-validator';

IsOptional; IsString;

// is fixed to this (note double space after brace):
import {  IsOptional, IsString } from 'class-validator';

IsOptional; IsString;
```

Example 2:
```typescript
// file:
import { IsInt, IsOptional, IsString } from 'class-validator';

IsInt; IsString;

// is fixed to this (note double space after IsInt):
import { IsInt,  IsString } from 'class-validator';

IsInt; IsString;
```

**Comment by sweepline** [comment]
I'll take a look at this and #7 at some time. I run an import sorting rule besides this at my job that fixes this and #7, so it isnt so high on my list of things at the moment.

**Comment by quezak** [comment]
@sweepline which sorting rule are you using?

A workaround for this issue with spaces is the `no-multi-spaces` rule autofixer, but it's not so simple for #7 with newlines. I'm using the `import/order` rule from [eslint-plugin-import](https://github.com/benmosher/eslint-plugin-import/), with import grouping configured. It does fix undeleted empty lines within an import group, but it doesn't fix empty lines on the top of the file and double empty lines between groups (I have `no-multiple-empty-lines` and `import/newline-after-import` set to 2, an old Python habit :smiley: ).

**Comment by krzkaczor** [comment]
@quezak check out this sorting rule: https://github.com/lydell/eslint-plugin-simple-import-sort

**Comment by quezak** [comment]
thx @krzkaczor looks like this one is both simple and configurable enough :)

**Comment by sweepline** [comment]
I have fixed this, but decided that it should just remove the whitespace before the unused import. That means it wont fix if there is multiple spaces after the comma. Also it wont add in spaces if there is no space after the comma.
